Framework/src/runPostProcessingOCC.cxx

```diff
@@ -91,10 +91,10 @@ class PostProcessingOCCStateMachine : public RuntimeControlledObject
     try {
       mRunner->start();
     } catch (const std::exception& ex) {
-      ILOG(Error) << "Exception caught: " << ex.what() << ENDM;
+      ILOG << LogErrorSupport << "Exception caught: " << ex.what() << ENDM;
```

Comment:
Why some catch statements have the "support" severity, while others have "Ops"?

Reply:
Indeed, I have put all of them with Support